.github/workflows/deploy-storefrontcloud.yml

```diff
@@ -11,18 +11,23 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v1
-      - name: Add theme
+      - name: Add theme master
+        if: github.ref != 'refs/heads/develop'
```

Comment:
Shouldn't be `master`?

Reply:
nope, because we want use master branch from default theme for master and hotfix. I can change it to `github.ref == 'refs/heads/master' || contains(github.ref, 'hotfix/v')`